python/tskit/trees.py

```diff
@@ -7558,6 +7566,8 @@ def genetic_relatedness(
             sample sets (rather than segregating between all of the samples of
             the tree sequence).
         :return: A ndarray with shape equal to (num windows, num statistics).
+            If windows=None and there is one pair of sample sets, a numpy scalar is
+            returned.
```

Comment:
This message has a different order to the previous one (it's nice to have them identical for searching purposes)?